components/brave_wallet/common/brave_wallet.mojom

```diff
@@ -608,6 +611,12 @@ struct SwitchChainRequest {
   string chain_id;
 };
 
+struct GetEncryptionPublicKeyRequest {
+  url.mojom.Url origin;
+  string address;
+  string? message;
+};
+
```

Comment:
note this is just renamed and moved from `struct EncryptionKeyRequest` which I think @Douglashdaniel added. I plan to move this into 2 diff request types, one for decrypt that uses message and this one that doesn't have `message` in the next PR.

Reply:
maybe add this comment link into the next issue you will open PR for?

Reply:
done

Reply:
https://github.com/brave/brave-core/pull/12517/files#diff-9f2cf11d57c8efee5461b92cdb38fa327b70c25fd8d009f7692049e8e88944a8R598